release_notes/Service_Fabric_ReleaseNotes_10.md

```diff
@@ -0,0 +1,75 @@
+|**Windows - 10.0.1816.9590<br>Ubuntu 18 - 10.0.1728.1<br>Ubuntu 20 - 10.0.1728.1** | **Feature** | Extensible Storage Engine(ESE) | **Brief Description**: The legacy auto-compaction settings, "CompactionThresholdInMB", "FreePageSizeThresholdInMB", and "CompactionProbabilityInPercent", has been retired and will no longer be used for determining auto-compaction at open. Instead, a new setting, "FreePageSizeThresholdInPercent", introduced in versions 9.1CU3 and 9.0CU9, will be used for determining auto-compaction at open. <br> **Breaking Change**: Any pre-existing overrides of the legacy auto-compaction settings will be dismissed. <br> **WorkAround**: To fine-tune the frequency and conditions of auto-compaction, kindly utilize the "FreePageSizeThresholdInPercent" setting. <br>**Documentation Reference**: Read more about it at - <ul><li>[FreePageSizeThresholdInPercent](https://learn.microsoft.com/en-us/dotnet/api/system.fabric.localesestoresettings.freepagesizethresholdinpercent?view=azure-dotnet)</li></ul>
+|**Windows - 10.0.1816.9590<br>Ubuntu 18 - 10.0.1728.1<br>Ubuntu 20 - 10.0.1728.1** | **Feature** | Extensible Storage Engine(ESE) | **Brief Description**: The legacy defragmentation setting, "DefragThresholdInMB," has been phased out for determining defragmentation occurrences. Instead, regular defragmentation will be triggered, and its frequency can be fine-tuned using the "MaxDefragFrequencyInMinutes" setting. Regular defragmentation will take place independent of the database size, leading to more precise calculations for the physical and logical database size, critical for auto-compaction determinations. <br>  **Breaking Change**: Any existing overrides of the "DefragThresholdInMB" setting will be disregarded. <br> **WorkAround**: Adjust defragmentation frequency by utilizing the "MaxDefragFrequencyInMinutes" setting.<br>**Documentation Reference**: Read more about it at - <br> <ul><li>[MaxDefragFrequencyInMinutes](https://learn.microsoft.com/en-us/dotnet/api/system.fabric.localesestoresettings.maxdefragfrequencyinminutes?view=azure-dotnet) </ul></li> 
```

Comment:
instead of "leading to more precise calculations for the physical and logical database size, critical for auto-compaction determinations", say -
"leading to a more precise calculation for the physical and the logical database size, which is critical for auto-compaction determination"